package.json

```diff
@@ -112,6 +116,12 @@
     "test": "jest --coverage",
     "watch": "jest --watch"
   },
+  "lint-staged": {
+    "*.js": [
+      "prettier --write",
+      "git add"
```

Comment:
This breaks Git compatibility when using [`git add -p`](https://git-scm.com/docs/git-add#git-add--p) to stage commits

See https://github.com/stylelint/stylelint/pull/2706#issuecomment-312465826